.github/workflows/build.yml

```diff
@@ -0,0 +1,74 @@
+name: Build pglookout
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '**'
+  pull_request:
+
+jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # only use the newest version for the lint step -> as long as the other version can run it it's ok
+        python-version: [3.9]
+
+    steps:
+
+      - id: checkout-code
+        uses: actions/checkout@v2
+
+      - id: prepare-python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - id: dependencies
+        run: |
+          pip install -e .
+          pip install --upgrade pylint flake8 pytest mock
```

Comment:
We probably should pin the version for these packages if we don't want the build to suddenly fail without any code change.

Reply:
Currently I would leave it as is: this should install the latest, if it fails, I would rather fix it than not know about it...

Reply:
Ok, this repo is fairly small so it shouldn't be too long to fix new errors when they arrive.